README.md

```diff
@@ -16,38 +18,29 @@ Build and push all new version of the gem:
 
 ```yaml
     steps:
-    # Setup ruby environment
     - uses: actions/checkout@v2
     - uses: ruby/setup-ruby@v1 # .ruby-version
       with:
-        bundler-cache: true # bundle install and cache
+        bundler-cache: true    # bundle install
+    - run: bundle exec rake build
 
-    - name: Build Gem
-      shell: bash
-      run: bundle exec rake build
-
-    - name: Setup GPR
-      uses: fac/ruby-gem-setup-github-packages-action@v1
+    - uses: fac/ruby-gem-setup-github-packages-action@v2
       with:
         token: ${{ secrets.github_token }}
 
-    - name: Push Gem
-      uses: fac/ruby-gem-push-action@v1
+    - uses: fac/ruby-gem-push-action@v2
+      with:
+        key: github
 ```
 
-If you want to use a different gem host or key:
-
-```yaml
-    - name: Push Gem
-      uses: fac/ruby-gem-push-action@v1
-      env:
-         gem_host: http://gems.example.com
-         gem_host_api_key: ${{ secrets.EXAMPLE_API_KEY }}
-```
+Note, that the ruby-gem-push-action, will push to the host given in the gemspec. The token needs to match. Trying to push to a different host will fail.
 
 ### Separate release and pre-release workflow
 
-You probably don't want to push all versions from any branch. More likely you would want to push release versions from your default branch (e.g. main) and pre-release from PR builds. To help with this the release and pre-release inputs can be used:
+
+By the default, the action releases, (er!) release versions, that is, non pre-release versions, those are skipped over with a message. Setting the input `pre-release: true`, reverses that, releasing new pre-release versions and skipping over live releases. That way, you can have 2 calls to the action, using the workflow to decide the logic.
```

Comment:
drop the `the`, `By default`. Can switch `releases` for `publishes` or plain old `pushes`. And can simplify the caveat:

> By default, the action only acts on non-pre-release versions, and prints a message if it thinks the gem has a pre-release version number. If you set the input option `pre-release: true`, then it will only act on pre-release versions, and will skip over regular versions. That way, [… no change…]